application/associations.py

```diff
@@ -263,7 +263,10 @@ def edit_file(self, selection):
 			self._application.create_terminal_tab(active_object._notebook, options)
 
 		else:
-			subprocess.Popen(split_command, cwd=os.path.dirname(selection[0]))
+			cwd = None
```

Comment:
Am not liking this approach. Just downright butchering string, we need a more generic approach where we won't have to revisit this issue in the future should there be a protocol that does something differently.